deployment/sandbox-v2/README.md

```diff
@@ -21,9 +21,9 @@ For reference, the sandbox architecture is depicted below:
 
 # Setting up the Sandbox Architecture
```

Comment:
Could we add "on Azure" to the title here? just to make it clearer that we've specialised the instructions